src/screencast/screencast.c

```diff
@@ -419,7 +419,7 @@ static int method_screencast_start(sd_bus_message *msg, void *data,
 		start_screencast(cast);
 	}
 
-	while (cast->node_id == 0) {
+	while ((int)cast->node_id == -1) {
```

Comment:
It seems like the node ID is initialized to `SPA_ID_INVALID` in PipeWire. Can we use this value instead of the cast and hardcoded int?

Reply:
Makes sense!